test/e2e/e2e_test.go

```diff
@@ -42,7 +42,7 @@ var (
 
 func TestEndToEnd(t *testing.T) {
 	RegisterFailHandler(Fail)
-	SetDefaultEventuallyTimeout(1 * time.Minute)
+	SetDefaultEventuallyTimeout(5 * time.Minute)
```

Comment:
I mentioned this in one of the other PRs, but my preference is to keep the default timeout at 1 minute and make exceptions if there's a good reason to do it on a case-by-case basis.